src/Containers.jsx

```diff
@@ -176,13 +177,11 @@ class Containers extends React.Component {
         const columnTitles = [_("Name"), _("Image"), _("Command"), _("CPU"), _("Memory"), _("State")];
         // TODO: emptyCaption
         let emptyCaption = _("No running containers");
-        const renderRow = this.renderRow;
         const containersStats = this.props.containersStats;
         // TODO: check filter text
-        let filtered = this.props.containers.filter(container => (!this.props.onlyShowRunning || container.State.Running));
-        let rows = filtered.map(function (container) {
-            return renderRow(containersStats, container);
-        }, this);
+        let filtered = [];
+        Object.keys(this.props.containers).filter(id => { if (!this.props.onlyShowRunning || this.props.containers[id].State.Running) { filtered.push(this.props.containers[id]) } });
```

Comment:
This works, but it's more elegant/simple to actually use the return value of `filter()`, as the original code did. I. e.

    let filtered = Object.keys(this.props.containers).filter(id => { if (!this.props.onlyShowRunning || this.props.containers[id].State.Running));

This applies to Images.jsx as well.